src/gg.regression.unity.bots/Runtime/Scripts/ClientDashboard/RGTcpManager.cs

```diff
@@ -0,0 +1,245 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net.Sockets;
+using JetBrains.Annotations;
+using RegressionGames.RemoteOrchestration.Models;
+using RegressionGames.RemoteOrchestration.Types;
+using RegressionGames.StateRecorder;
+using RegressionGames.StateRecorder.BotSegments;
+using RegressionGames.StateRecorder.BotSegments.Models;
+using UnityEditor;
```

Comment:
We're going to have issues in built games by always including UnityEditor 

see my other comment about how we may want to split this up